textproc/lilypond/Portfile

```diff
@@ -163,17 +163,12 @@ if {![variant_isset mactex]} {
 }
 
 configure.env-append    GUILE=${prefix}/bin/guile18
-configure.env-append    GUILE_CONFIG=${prefix}/bin/guile18-config
-configure.env-append    GUILE_TOOLS=${prefix}/bin/guile18-tools
+configure.env-append    GUILE_FLAVOR=guile-1.8
 
 configure.env-append    PYTHON=${configure.python}
 build.env-append        PYTHON=${configure.python}
 destroot.env-append     PYTHON=${configure.python}
 
-configure.env-append    PYTHON_CONFIG=${configure.python}-config
```

Comment:
I'm a little surprised we no longer need to manually specify MacPorts' python to guarantee a compatible version exists.